### PR TITLE
feat(grammar): Expose wildcard imports as named nodes

### PR DESCRIPTION
**Pull request by ValdezFOmar**
Parse "." and "*" for wildcard imports as separate nodes, allowing them to be more precisely highlighted.
Create a new named node `wildcard_import`.

### CODE REVIEW

**Comment by ValdezFOmar** [comment]
I've only notice now that a similar pull request was opened (https://github.com/fwcd/tree-sitter-kotlin/pull/122). But something different is that my implementation exposes `.` and `*` as separate nodes instead of being only one, which will allow more precise highlighting.
The use case for this is in nvim-treesitter (https://github.com/nvim-treesitter/nvim-treesitter/pull/6979) where we are trying to highlight all wildcard imports (`*` specifically) the same. Please consider merging my PR instead :pray:

File: grammar.js [hunk 1]
```diff
@@ -167,11 +167,13 @@ module.exports = grammar({
 
     import_header: $ => seq(
       "import",
-      $.identifier,
-      optional(choice(seq(".*"), $.import_alias)),
```
Comment (clason) [inline grammar.js:171]: Note that this construct (single child `seq` rule) is currently broken on tree-sitter master, so this (or a similar) change is needed to keep being able to generate a compatible parser.

**Comment by clason** [comment]
@fwcd @VladimirMakaev what's the current maintenance situation with this parser? I see a number of (even necessary) maintenance PRs being left unmerged, and I worry that sooner or later this parser will become unusable for downstream like Neovim.

**Comment by fwcd** [comment]
> what's the current maintenance situation with this parser?

It's maintained, but not super actively (at least on my part), largely due to a lack of spare time and other important projects, at least on my end. PRs are always welcome though, smaller ones are likely to be merged more quickly.

**Comment by fwcd** [comment]
Please regenerate the parser on an x86_64 Linux machine: #92 The simplest way to do that is in CI using the "Regenerate Parser" action, which should be available in your fork too (if not, leave a note). Docker also works.

**Comment by clason** [comment]
That is not the issue; the problem is that the CI is using a different (outdated?) tree-sitter version.

**Comment by fwcd** [comment]
The CI should use the same version of tree-sitter as the package, that is 0.22.1, and I would strongly recommend using the `npm`-installed version of tree-sitter for local development too. We pin it, because the playground and other tools have to be kept in sync, otherwise it becomes hard to keep track of the different environments that everyone builds the grammar in.

Updating to 0.22.6 is planned, but, as mentioned above, we have to be a bit careful about not breaking e.g. the playground by just bumping the version:

- #121 
- #126

**Comment by ObserverOfTime** [comment]
> It's maintained, but not super actively (at least on my part), largely due to a lack of spare time and other important projects.

If you need help with maintenance, we'd be happy to adopt it under [tree-sitter-grammars](https://github.com/tree-sitter-grammars).

**Comment by fwcd** [comment]
That's very nice, thanks, but I still prefer to keep it here for now. I originally put a lot of work into writing this grammar and also maintain the packages under my name, so I'd like to keep the canonical version here for now. Feel free to fork it into your organization if that suits your workflow!

**Comment by ObserverOfTime** [comment]
I mean you will keep maintaining it and the packages just as before, but have more hands on deck.

**Comment by fwcd** [comment]
Thanks!

**Pull request merged**